src/main/java/mate/academy/model/ShoppingCard.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCard {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
```

Comment:
```suggestion
    @OneToMany
        @JoinTable(joinColumns =
            @JoinColumn(name = "shopping_cart_id"),
            inverseJoinColumns =
            @JoinColumn(name = "ticket_id"))
    private List<Ticket> tickets;
```

we can rename our columns